### PR TITLE
New index.html design & added docu links

### PR DESCRIPTION
**Pull request by theRealImy**
The new index page looks like:
Screenshot 2021-11-09 at 16 17 49

After logged in a message with the WebID is displayed under the header, like:
grafik

Remark: this is from the template so please do not mind the {{..}} fields they will be replaced by pod providers.

### CODE REVIEW

**Comment by ewingson** [comment]
@theRealImy @jeff-zucker @bourgeoa 

I'm not sure what will happen without the removed javascript-code in the `index.html`
we will have to put hands *manually* to the file, or did I misunderstand anything and the template will be processed in any way ?

in the suggested changes/PR I'm afraid we will lose the functionality of the two buttons. but the impression of the look&feel is great. correct me if I'm too naive.

**Comment by ewingson** [comment]
everything back, the code is now in common/js/index-buttons.js as I just see.......

**Review by ewingson — COMMENTED** [review]
LGTM

**Comment by bourgeoa** [comment]
@theRealImy 
Seems nice. I must test it.

I like the WebId link appearing under the header.

A few more comments.  Your changes removes the no-brand concept :
- this a generic index.html and nothing specific should be there
- anyone could have his own brand logo including in the header
- links should be minimal : your additions give too much focus on solid
- remove the link to create your own server. It is not even on NSS but on CSS

File: common/js/index-buttons.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    else {
+        let webId = window.localStorage.getItem(keyname);
+
+        // IF NOT LOGGED IN AND COOKIE EXISTS: REMOVE COOKIE, HIDE WELCOME, SHOW LINK TO PROFILE
+        if( webId ) {
+        window.localStorage.removeItem(keyname);
+        //document.getElementById('prototype').style.display = "none";
+        document.getElementById('loggedIn').style.display = "block";
+        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is: <a href="${webId}">&lt;${webId}&gt</a>. Visit your profile to log into your Pod.</p>`;
```
Comment (bourgeoa) [inline common/js/index-buttons.js:22]:
```suggestion
        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>. Visit your profile to log into your Pod.</p>`;
```

**Comment by bourgeoa** [comment]
Trying locally it appears : 
- that some CSS have changed and now the register pane appear in too small characters.
- I also noticed that the displayed WebId has `<webId>`. We must remove the `<>`

**Comment by theRealImy** [comment]
I changed the texts and made the logo configurable. The page now looks like:
grafik
and after login:
grafik
I also fixed the problem of the font size on the register page.

**Review by bourgeoa — APPROVED** [review]
@timea I tried locally and there is problem. Did you miss to push ?

File: common/js/index-buttons.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        await UI.authn.authSession.logout();
+    }
+    else {
+        let webId = window.localStorage.getItem(keyname);
+
+        // IF NOT LOGGED IN AND COOKIE EXISTS: REMOVE COOKIE, HIDE WELCOME, SHOW LINK TO PROFILE
+        if( webId ) {
+        window.localStorage.removeItem(keyname);
+        document.getElementById('loggedIn').style.display = "block";
+        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>. Visit your profile to log into your Pod.</p>`;
```
Comment (TallTed) [inline common/js/index-buttons.js:22]: Will make the instruction clearer.
```suggestion
        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>.</p> <p>Visit your profile to log into your Pod.</p>`;
```

**Review by bourgeoa — APPROVED** [review]
LGTM

**Comment by ewingson** [comment]
personally I find a little margin in the dt element better. but that's just perfectionism and a bit opinionated.

**Pull request merged**